lib/validates_zipcode/validator.rb

```diff
@@ -27,7 +27,7 @@ def validate_each(record, attribute, value)
       alpha2 = @country_code || record.send(@country_code_attribute)
 
       unless ValidatesZipcode::Zipcode.new(zipcode: value.to_s, country_alpha2: alpha2).valid?
-        record.errors.add(attribute, I18n.t('errors.messages.invalid_zipcode', value: value, default: 'Zipcode is invalid'))
+        record.errors.add(attribute, :invalid_zipcode, message: I18n.t('errors.messages.invalid_zipcode', value: value, default: 'Zipcode is invalid'))
```

Comment:
Hmm I'm curious: wouldn't `message` be unnecessary then? 

`record.errors.add(attribute, :invalid_zipcode, value: value, default: 'Zipcode is invalid'))`